lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -2491,14 +2529,24 @@ def main():
                 set_vm_power_state(pyv.content, vm, 'poweredoff', module.params['force'])
             result = pyv.remove_vm(vm)
         elif module.params['state'] == 'present':
-            if module.check_mode:
-                result.update(
-                    vm_name=vm.name,
-                    changed=True,
-                    desired_operation='reconfigure_vm',
-                )
-                module.exit_json(**result)
-            result = pyv.reconfigure_vm()
+            if 'existing_vm' in module.params['customization'] and module.params['customization']['existing_vm']:
```

Comment:
This means (in combination with line 2541-2549), that a VM can not be reconfigured and customized at the same time with this change?

Reply:
Yeah, what I intended to do is "customize" and "re-configure" are two separate operations. You mean do "re-configure" first and then do "customize" on this VM? Yeah, that's also a scenario.

Reply:
yeah, indeed... if it's not supported, it should probably be mentioned in the documentation?

Reply:
Ok, thank you very much :), I'll make some changes on this later.

Reply:
I moved the "customize existing vm" execution to the last part of "reconfigure vm".